src/Utilities/useVerifyName.js

```diff
@@ -5,6 +5,7 @@ export const useVerifyName = (name, remediationsList) => {
   const [isDisabled, setIsDisabled] = useState(false);
   const mounted = useRef(false);
   const timerRef = useRef(null);
+  const playbookNamePattern = /^(?!$).*[\w\d]+.*$/;
```

Comment:
The regex used on the api end is `^[^\s].*[^\s]$` - so basically anything as long as it doesn't start or end with whitespace.  Also: \w includes [0-9] so the \d isn't doing anything here.

It's ok if this is more restrictive than the api, though.

Reply:
this is the pattern baked in from a while back, ill change it to match the api though, i think that makes more sense